### ISSUE TITLE
No INSTALL file after `autoreconf --install`

### ISSUE THREAD

**Issue by cibersheep** [jxself/zilutils#1]
Hello, 
In the main readme of the project says: "[...] first run autoreconf --install; then see the INSTALL file for installation information." but after runt he command there is no INSTAL file.

This is the output I get
```
$ autoreconf --install

configure.ac:30: installing './compile'
configure.ac:26: installing './install-sh'
configure.ac:26: installing './missing'
Makefile.am: installing './depcomp'
configure.ac:30: installing './compile'
configure.ac:26: installing './install-sh'
configure.ac:26: installing './missing'
Makefile.am: installing './depcomp'
configure.ac: installing './ylwrap'
configure.ac:24: installing './install-sh'
configure.ac:24: installing './missing'
```

And this is the result
```
$ ls 
aclocal.m4      configure     COPYING     Makefile.am  missing    zilasm
autom4te.cache  configure.ac  install-sh  Makefile.in  README.md  zilc
```

I'm on Ubuntu 16.04

Thank you

**Comment by jxself** [comment]
I believe that the bug you reported is now fixed with commit d56c703. If so please feel free to close this as being resolved.

**Comment by cibersheep** [comment]
Awesome. Thank you
